nfs-ganesha/build/build_deb

```diff
@@ -58,8 +60,10 @@ sudo pbuilder --clean
 
 mkdir -p $WORKSPACE/dist/deb
 
+# use libcephfs and librgw from shaman
 REPO_URL="https://shaman.ceph.com/api/repos/ceph/$CEPH_BRANCH/$CEPH_SHA1/$DISTRO/$DIST/repo"
 echo "EXTRAPACKAGES=\"libcephfs-dev\"" >> ~/.pbuilderrc
+echo "EXTRAPACKAGES=\"librgw-dev\"" >> ~/.pbuilderrc
```

Comment:
I don't think this will work because it is redefining `EXTRAPACKAGES` from the line before. For multiple packages you need a space separated string like:

    EXTRAPACKAGES="libcephfs-dev librgw-dev"